01-setting.tex

```diff
@@ -557,7 +557,7 @@ \subsection*{Proofs of Security}
 protocol, except with negligible probability.
 
 The proofs of these theorems will take the form of a \emph{computational reduction},
-and they will look roughly as follows, when we are designing a new protocol $\Pi^*$:
+and they will look roughly as follows, when we are designing a new protocol $\Pi$:
```

Comment:
I think the original is correct here. Am I missing something?

Reply:
In that case, I might have misunderstood the structure of a computational reduction a bit. It seemed to me that the "new" protocol was Pi, since Pi was being built on top of Pi*. An example that I thought followed this was the reduction on pg 122 about merkle trees: we were proving that a MT built on top of a CRHF was secure. Here, it felt like MT was the "new" protocol, and we were going to show that breaking MT would break CRHF, so MT is unbreakable. So, I assumed the "new" protocol was just the one that got it's security from something that we already assumed was secure.